catalog/services_state.go

```diff
@@ -58,8 +59,8 @@ func NewServer(name string) *Server {
 		Name: name,
 		// Pre-create for 5 services per host
 		Services:    make(map[string]*service.Service, 5),
-		LastUpdated: time.Unix(0, 0),
-		LastChanged: time.Unix(0, 0),
+		LastUpdated: time.Time{},
```

Comment:
I'd rather not make this change. I definitely have outside code that uses this in a few projects.

Reply:
If we can back this out, I'll merge it!

Reply:
Done and created #54 to keep a record of this issue in case we want to fix it in the future.